server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -508,6 +524,14 @@ private static boolean seedsChanged(final List<String> oldSeedNodes, final List<
         return oldSeeds.equals(newSeeds) == false;
     }
 
+    private static boolean authorizationChanged(String oldAuthorization, String newAuthorization) {
```

Comment:
Since we are not authorizing the connection itself, is there a need to rebuild the connection on changes ? i.e. I think we can omit authorization from the rebuild checks here, especially if the setting is moved.

Reply:
I think it can be moved and checked in `RemoteConnectionStrategy.shouldRebuildConnection`.

Reply:
Moved to RemoteConnectionStrategy.

Reply:
Removed authorization from both the profile and strategy classes.